subworkflows/local/standardisation_profiles.nf

```diff
@@ -70,7 +70,11 @@ workflow STANDARDISATION_PROFILES {
 
 
     TAXPASTA_MERGE       (ch_input_for_taxpasta.merge      , ch_taxpasta_tax_dir, [])
+    ch_versions = ch_versions.mix( TAXPASTA_MERGE.out.versions )
     TAXPASTA_STANDARDISE (ch_input_for_taxpasta.standardise, ch_taxpasta_tax_dir    )
+    ch_version = ch_versions.mix( TAXPASTA_STANDARDISE.out.versions )
```

Comment:
Do we still only use one element here? Or does that come later?

```suggestion
    ch_versions = ch_versions.mix( TAXPASTA_MERGE.out.versions.first() )
    TAXPASTA_STANDARDISE (ch_input_for_taxpasta.standardise, ch_taxpasta_tax_dir    )
    ch_version = ch_versions.mix( TAXPASTA_STANDARDISE.out.versions.first() )
```

Reply:
That is a good point, I do not think it comes later.